### ISSUE TITLE
Why we need the is_training flag

### ISSUE THREAD

**Issue by Uoops** [YunYang1994/tensorflow-yolov3#59]
I do not think the is_training does anything here. Why this flag is necessary?

**Comment by YunYang1994** [comment]
for batch-normalization computation
